test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -2193,6 +2193,161 @@ Describe "Invoke-WebRequest tests" -Tags "Feature", "RequireAdminOnWindows" {
             $pathologicalRatio | Should -BeGreaterThan 5
         }
     }
+
+    Context 'WebSession persistence tests' {
+        $connWarning = 'WebSession properties changed: new Http connection will be required'
+
+        It 'Connection persistence maintained' {
+            $Uri = Get-WebListenerUrl
+            $null = Invoke-WebRequest -Uri $uri -SessionVariable Session -wv warnings
```

Comment:
```suggestion
            $null = Invoke-WebRequest -Uri $Uri -SessionVariable Session -WarningVariable Warnings
```
Could you please, always use variables with the same capitalization in the tests